static/js/components/dialogs/CollectionFormDialog.js

```diff
@@ -76,28 +78,67 @@ class CollectionFormDialog extends React.Component {
     dispatch(uiActions.setAdminLists(event.target.value));
   };
 
+  validate = () => {
```

Comment:
This function looks like it's unused, can you remove it?

Reply:
OK